packages/storybook/config/preview.tsx

```diff
@@ -1,7 +1,11 @@
-// import '@nl-design-system-unstable/voorbeeld-design-tokens/src/font.js';
-import '@nl-design-system-unstable/voorbeeld-design-tokens/dist/index.css';
+/* eslint-env node */
+// import '@kernteam/design-tokens/dist/index.css';
+// import '@kernteam/font/src/index.scss';
 import { defineCustomElements } from '@kernteam/web-components-stencil/loader/index.js';
 import type { Preview, StoryContext } from '@storybook/react';
+import prettierBabel from 'prettier/parser-babel';
+import prettier from 'prettier/standalone';
+import * as ReactDOMServer from 'react-dom/server';
```

Comment:
🤔 zou `import { renderToStaticMarkup } from 'react-dom/server';` ook werken? Wel simpeler en netter, minder dependencies.

Reply:
ReactDOMServer word direct aangeroepen in het bestand, dus kan niet simpeler